cmd/snap/cmd_first_boot.go

```diff
@@ -41,5 +39,9 @@ func (x *cmdInternalFirstBoot) Execute(args []string) error {
 		return ErrExtraArgs
 	}
 
-	return boot.FirstBoot()
+	println(`
+This command does nothing, its only there because we can not update files 
+in the writable space after they got copied out of the snap.
```

Comment:
Would be nice to perhaps do:
```
fmt.Fprintf(Stderr, "firstboot command is deprecated")
```
and comment more clearly about the circumstances which makes us keep this in place, so that we won't forget either to keep it or to remove it, once we feel it's time to do that.